atr/shared/distribution.py

```diff
@@ -17,74 +17,125 @@
 import atr.models.distribution as distribution
 import atr.models.sql as sql
-import atr.storage as storage
-import atr.template as template
 import atr.util as util
 
 type Phase = Literal["COMPOSE", "VOTE", "FINISH"]
 
 
-class DeleteForm(forms.Typed):
-    release_name = forms.hidden()
-    platform = forms.hidden()
-    owner_namespace = forms.hidden()
-    package = forms.hidden()
-    version = forms.hidden()
-    submit = forms.submit("Delete")
-
-
-class DistributeForm(forms.Typed):
-    platform = forms.select("Platform", choices=sql.DistributionPlatform)
-    owner_namespace = forms.optional(
+class DistributionPlatformWrapper(enum.Enum):
+    """Wrapper enum for distribution platforms."""
+
+    ARTIFACT_HUB = "Artifact Hub"
+    DOCKER_HUB = "Docker Hub"
+    MAVEN = "Maven Central"
+    NPM = "npm"
+    NPM_SCOPED = "npm (scoped)"
+    PYPI = "PyPI"
+
+
+def wrapper_to_sql(wrapper: DistributionPlatformWrapper) -> sql.DistributionPlatform:
+    """Convert wrapper enum to SQL enum."""
+    match wrapper:
+        case DistributionPlatformWrapper.ARTIFACT_HUB:
+            return sql.DistributionPlatform.ARTIFACT_HUB
+        case DistributionPlatformWrapper.DOCKER_HUB:
+            return sql.DistributionPlatform.DOCKER_HUB
+        case DistributionPlatformWrapper.MAVEN:
+            return sql.DistributionPlatform.MAVEN
+        case DistributionPlatformWrapper.NPM:
+            return sql.DistributionPlatform.NPM
+        case DistributionPlatformWrapper.NPM_SCOPED:
+            return sql.DistributionPlatform.NPM_SCOPED
+        case DistributionPlatformWrapper.PYPI:
+            return sql.DistributionPlatform.PYPI
+
+
+def sql_to_wrapper(platform: sql.DistributionPlatform) -> DistributionPlatformWrapper:
+    """Convert SQL enum to wrapper enum."""
+    match platform:
+        case sql.DistributionPlatform.ARTIFACT_HUB:
+            return DistributionPlatformWrapper.ARTIFACT_HUB
+        case sql.DistributionPlatform.DOCKER_HUB:
+            return DistributionPlatformWrapper.DOCKER_HUB
+        case sql.DistributionPlatform.MAVEN:
+            return DistributionPlatformWrapper.MAVEN
+        case sql.DistributionPlatform.NPM:
+            return DistributionPlatformWrapper.NPM
+        case sql.DistributionPlatform.NPM_SCOPED:
+            return DistributionPlatformWrapper.NPM_SCOPED
+        case sql.DistributionPlatform.PYPI:
+            return DistributionPlatformWrapper.PYPI
+
+
+class DeleteForm(form.Form):
+    release_name: str = form.label("Release name", widget=form.Widget.HIDDEN)
+    platform: str = form.label("Platform", widget=form.Widget.HIDDEN)
+    owner_namespace: str = form.label("Owner namespace", widget=form.Widget.HIDDEN)
+    package: str = form.label("Package", widget=form.Widget.HIDDEN)
+    version: str = form.label("Version", widget=form.Widget.HIDDEN)
+
+
+class DistributeForm(form.Form):
+    platform: form.Enum[DistributionPlatformWrapper] = form.label(description="Platform")
```

Comment:
We don't use keywords with `form.label`. I guess the wider convention is that we don't use keywords for arguments before `*`, but I'm not sure how widely that holds and if it's really a worthwhile convention.